src/features/tasks/tasksSlice.js

```diff
@@ -359,6 +457,27 @@ export const fetchTask = (taskId) => {
               },
               FAIL: (res) => dispatch(updateCameraSerialNumberFailure(res)),
             },
+            DeleteImages: {
+              COMPLETE: (res) => {
+                dispatch(deleteImagesSuccess(res.task.output.imageIds));
+              },
+              FAIL: (res) => dispatch(deleteImagesFailure(res))
+            },
+            DeleteImagesByFilter: {
+              COMPLETE: (res) => {
+                dispatch(deleteImagesByFilterSuccess());
+                dispatch(
+                  setFocus({
+                    index: { image: null, object: null, label: null },
+                    type: 'auto',
+                  }),
+                );
+                dispatch(setModalOpen(false));
+                dispatch(setModalContent(null));
+                dispatch(fetchImages(res.task.output.filters));
```

Comment:
Why do we `fetchImages` after a successful `DeleteImagesByFilter` task but not `DeleteImages` task?

Reply:
We talked about this on huddle, but this is because when deleting by selection, we currently just remove those ImageIds from the workingImages set, and now actually checking if the images are removed. But when deleting by filters, we do not necessarily have the full list of image ids so something similar can't necessarily be done. One option is to just set the workingImages set to an empty list since that in theory should be the final result